src/bin/sifis-generate.rs

```diff
@@ -84,12 +162,32 @@ enum Cmd {
     Yarn(CommonData),
 }
 
+fn local_config() -> anyhow::Result<PathBuf> {
+    let config_dir = std::env::var("XDG_CONFIG_HOME")
```

Comment:
Is this environment variable usable on Windows?

Reply:
as long you set it.